lxd/instance/drivers/driver_qemu.go

```diff
@@ -4254,23 +4291,24 @@ func (vm *qemu) UpdateBackupFile() error {
 	return pool.UpdateInstanceBackupFile(vm, nil)
 }
 
-func (vm *qemu) cpuTopology(limit string) (int, int, int, map[uint64]uint64, error) {
+func (vm *qemu) cpuTopology(limit string) (int, int, int, map[uint64]uint64, map[uint64][]uint64, error) {
```

Comment:
@stgraber please could you add a comment to this function explaining what the return values are, as the function signature itself doesn't give much away. Thanks